analyzers/src/SonarAnalyzer.Common/Helpers/SymbolHelper.cs

```diff
@@ -209,6 +209,34 @@ public static bool IsPubliclyAccessible(this ISymbol symbol)
             symbol?.GetAttributes().Where(a => a.AttributeClass.IsAny(attributeTypes))
             ?? Enumerable.Empty<AttributeData>();
 
+        internal static IEnumerable<AttributeData> GetAttributesWithInherited(this ISymbol symbol)
```

Comment:
The class is already internal.
```suggestion
        public static IEnumerable<AttributeData> GetAttributesWithInherited(this ISymbol symbol)
```